Projects/GLU/GLU01/GLU01-AWS100.md

```diff
@@ -0,0 +1,38 @@
+# GLU02-AWS100 - Introduction to SNS (Simple Notification Service)
+
+## Cloud Service Provider
+- Amazon Web Services
+
+## Difficulty
+- Level 100 (Introductory)
+
+## Project's Author(s)
+
+- [Edward Allen Mercado](https://twitter.com/edwardmercado_)
+
+## Objectives
+
+### You need to complete the following:
+
+- Create a SNS Topic
+- Subscribe to a SNS Topic
+- Receive email notification from the SNS Topic created
```

Comment:
Need a bit more clarity on the objectives since it's Level 100.
- Create an SNS topic
- Subscribe to that SNS topic with an email adress of yours
- Send a test message through the SNS topic 
- Make sure you received the message to your email adress

Reply:
I see, I got it, updated them now on my latest commits